backfill/gormstore.go

```diff
@@ -195,19 +273,39 @@ func (j *Gormjob) SetState(ctx context.Context, state string) error {
 	return j.db.Save(j.dbj).Error
 }
 
-func (j *Gormjob) FlushBufferedOps(ctx context.Context, fn func(kind, path string, rec *typegen.CBORMarshaler, cid *cid.Cid) error) error {
+var ErrEventGap = fmt.Errorf("buffered event revs did not line up")
+
+func (j *Gormjob) FlushBufferedOps(ctx context.Context, fn func(kind, path string, rec typegen.CBORMarshaler, cid *cid.Cid) error) error {
```

Comment:
How does this work with deletes? IIRC the `rec` being a pointer worked with deletes because it'd be `nil` when deleting a record, does it just end up being an empty `typegen.CBORMarshaler` in this case?